scripts/build_frontend.sh

```diff
@@ -5,6 +5,8 @@ source scripts/helpers.sh
 
 pnpm_required
 
+python scripts/generate_theme.py
```

Comment:
I'm not totally against it for now but I actually want us to get rid of as many of these bash scripts as we can. Having multiple ways to run scripts is confusing a pain to  maintain. I'd like us to use python scripts directly or pnpm. 

Since we already have a way of running all of our scripts without the indirection of additional bash scripts (which only work in certain environments anyway).